src/Apps/Artwork/Components/ArtworkImageBrowser/index.tsx

```diff
@@ -46,17 +46,10 @@ export const ArtworkImageBrowserFragmentContainer = createFragmentContainer<
 >(ArtworkImageBrowserContainer, {
   artwork: graphql`
     fragment ArtworkImageBrowser_artwork on Artwork {
-      title
       image_alt: to_s
-      image_title
-      href
-
       ...ArtworkActions_artwork
       image {
         id
-        url(version: "larger")
```

Comment:
I think that these iare actually used by force for the View in Room feature (see usage [here](https://github.com/artsy/force/blob/master/src/desktop/apps/artwork/client.tsx#L86), passed in [here](https://github.com/artsy/reaction/blob/master/src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx#L106))

But actually, in general we'll need to make sure that we're not removing fields that are necessary for hooks within force/other parent apps.

Reply:
> But actually, in general we'll need to make sure that we're not removing fields that are necessary for hooks within force/other parent apps.

Absolutely, I'm keeping that in mind. I think in this case we're safe. Unless I'm missing something, this image object doesn't get passed anywhere else, even transitively, and it's url, height, and width are not referenced anywhere.

Reply:
I believe they're referenced from force when they are passed in [via the `mediator`](https://github.com/artsy/reaction/blob/master/src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx#L114) component.

Reply:
Oh oh, but I guess in this case they are included in the `ArtworkActions` fragment container (missed that!).

Reply:
Right, but ArtworkActions has it's own fragment where it selects these fields https://github.com/artsy/reaction/blob/master/src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx#L286